crates/criterion_compat/src/compat/group.rs

```diff
@@ -79,7 +79,7 @@ impl<'a, M: Measurement> BenchmarkGroup<'a, M> {
 
 // Dummy methods
 #[allow(unused_variables)]
-impl<'a, M: Measurement> BenchmarkGroup<'a, M> {
+impl<M: Measurement> BenchmarkGroup<'_, M> {
```

Comment:
As we discussed, it would be nice to revert those lifetime changes since those might be breaking